### ISSUE TITLE
Focus "erase browsing" notification remains when opening links in focus

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/focus-android#790]
STR
1. Open a link from another app in focus
1. hit back

Expected: "erase browsing" notification clears
Actual: notification remains

This is on my personal GS5 with the Play Store build.

**Comment by pocmo** [comment]
The notification does not clear because your history is not erased and the Focus window with the website open is still open and you can switch to it via the "recent apps" screen. This was a deliberate design choice.

**Comment by mcomella** [comment]
But if I hit back, we go back in the back stack so the next time I open the browser, I open on the "Search or enter address" home page, rather than anything related to the page I was on. Since there's no forward button either, I don't see a reason to keep the browsing session alive.

**Comment by damosullivan** [comment]
Came to report this too. I believe the Back button should erase browsing, or it should at lease be an option.

**Comment by pocmo** [comment]
> But if I hit back, we go back in the back stack so the next time I open the browser, I open on the "Search or enter address" home page, rather than anything related to the page I was on.

This is the actual bug then. You used to get back to the page (and that's what we intended to do).

**Comment by mcomella** [comment]
> You used to get back to the page (and that's what we intended to do).

fwiw, I find this makes Focus annoying to use as a default browser – every time I click a link and close Focus via the back button, I have to take an additional step to drag down the notification bar and click, "Erase browsing history" (or suffer with 1) potentially private content open or 2) an annoying notification all the time.

**Comment by pocmo** [comment]
I discussed this for some time with antlam before the first release and that's the flow we settled on. But I agree that it gets annoying when using Focus as a default browser - whatever you do there are always two steps needed to get back and erase. I often end up with a notification in the drawer and it feels like overhead to clear it. Let's re-discuss this. I filed #888 and set the "needs triage" label.
